src/index.md

```diff
@@ -15,15 +15,17 @@ Get started easily, get more done, and never play twenty questions with <code>ku
 <a href="{{site.docsurl}}install.html" class="brandButton u-marginBottomUnit u-marginRightUnit">
   {% include brandButtonBg.html %}
   <div class="buttonLabel brandButton-text">
-    Try Tilt Today
+    Try Tilt Now
   </div>
 </a>
 
-<h3 class="ctaLink u-marginBottomUnit">
-  <a href="https://github.com/windmilleng/tilt">
-    &hairsp;&gt; Check us out on GitHub&hairsp;
-  </a>
-</h3>
+<div class="flexblock u-marginTop1_25 u-marginBottom1_5">
```

Comment:
this removes the main github call to action. is that ok?

(AFAIK, the github call to action does get a lot of clicks. I suspect a certain type of person looks for the github link first, for a certain kind of social proof / credibility)

Reply:
Also, this additional div `<div class="flexblock u-marginTop1_25 u-marginBottom1_5">` breaks the layout of the page, since you don't close with `</div>`. If you remove this line, the two button links will sit inside the single parent div opened on line 14.